tests/out/spv/image.spvasm

```diff
@@ -1,19 +1,19 @@
 ; SPIR-V
 ; Version: 1.1
 ; Generator: rspirv
-; Bound: 174
+; Bound: 195
```

Comment:
ok, unfortunately this is sort of a blocker. We are increasing the ID count by 10% here without changing any functionality. We need to find a way to avoid this.
If I understand correctly, this PR is currently doing `OpLoad` on demand for globals and parameters.
What if we do it at the function start instead?

Reply:
Done, now the everything is loaded ahead of time